.github/workflows/errors.yml

```diff
@@ -0,0 +1,55 @@
+on: push
+
+jobs:
+  verify:
+    name: "Verify errors"
+    runs-on: ubuntu-latest
+    steps:
+      # This step checks out a copy of your repository.
+      - uses: actions/checkout@v2
+
+      # run checks on a series of directories
+      - name: Trying to run in a non-existing directory should fail
+        uses: citation-file-format/cffconvert-github-action@tests
+        with:
+          FAILURE_EXPECTED: '1'
+          WORKDIR: non-existing-directory/
+
+      - name: A directory with missing CITATION.cff file should fail
+        uses: citation-file-format/cffconvert-github-action@tests
+        with:
+          FAILURE_EXPECTED: '1'
+          WORKDIR: tests/cff-missing/
+
+      - name: A directory with missing zenodo file should fail
+        uses: citation-file-format/cffconvert-github-action@tests
+        with:
+          FAILURE_EXPECTED: '1'
+          WORKDIR: tests/zenodo-missing/
+
+      - name: A zenodo file with missing title field should fail
+        uses: citation-file-format/cffconvert-github-action@tests
+        with:
+          FAILURE_EXPECTED: '1'
+          WORKDIR: tests/zenodo-not-equivalent/
+
+      - name: A zenodo file with extra spaces should pass when DIFF_IGNORE_WHITESPACE is undefined
+        uses: citation-file-format/cffconvert-github-action@tests
+        with:
+          DIFF_IGNORE_WHITESPACE: ''
+          FAILURE_EXPECTED: '0'
+          WORKDIR: tests/zenodo-with-extra-whitespace/
+
+      - name: A zenodo file with extra spaces should pass when DIFF_IGNORE_WHITESPACE is True
+        uses: citation-file-format/cffconvert-github-action@tests
+        with:
+          DIFF_IGNORE_WHITESPACE: '1'
+          FAILURE_EXPECTED: '0'
+          WORKDIR: tests/zenodo-with-extra-whitespace/
+
+      - name: A zenodo file with extra spaces should fail when DIFF_IGNORE_WHITESPACE is False
+        uses: citation-file-format/cffconvert-github-action@tests
+        with:
+          DIFF_IGNORE_WHITESPACE: '0'
+          FAILURE_EXPECTED: '1'
+          WORKDIR: tests/zenodo-with-extra-whitespace/
```

Comment:
It makes sense to check if one or more fields exist but do not match.

Reply:
Could you be more specific?